### DISCUSSION TITLE
AI agent to help developers build with DeepKit

### DISCUSSION THREAD

**Discussion by samyakkkk** [marcj/deepkit#585] [Ideas]
Hi @marcj , would it be helpful users if we train an AI agent on the docs, code of the repository + the issues?

It can help users building with DeepKit by providing tailored code by just providing their usecase. plmk your thought - and I'm happy to volunteer to build one.

For context: I am also building a small [tool](https://commanddash.io/) for this problem.

**Comment by marcj** [comment]
I think it depends on how good the bot will be. Usually they are not that good and beginners are also not our target group, that's why we don't have one. If you can build one that is rather strong and even helpful in more complex/low-documented stuff, please feel free to build one. btw we also have lots of knowledge in discord (older stuff obsolete)

**Reply by samyakkkk** [reply]
I agree. the main challenge is chatbots giving out methods that doesn't exist. + unable to answer undocumented stuff.

these are the two main challenges along with the ease of access (in the IDE) we are working to solve. I'll share an agent to try here shortly, it's not perfect but we are zeroing in on the problem.
